src/client/stylesheets/style.less

```diff
@@ -283,8 +283,15 @@ a.contact-text:visited {
 }
 
 .deleted {
+	text-decoration: @brand-danger line-through;
 	text-decoration-line: line-through;
 	text-decoration-color: @brand-danger;
+	-webkit-text-decoration-line: line-through;
+	-webkit-text-decoration-color: @brand-danger;
+	-moz-text-decoration-line: line-through;
+	-moz-text-decoration-color: @brand-danger;
+	-ms-text-decoration-line: line-through;
+	-ms-text-decoration-color: @brand-danger;
```

Comment:
I am not finding any documentation for this prefix. Does this actually exist for microsoft browsers?

Reply:
yes **ms** is a valid prefix https://bitsofco.de/css-vendor-prefixes/

Reply:
yes, I do know it :)
But looking at the [support table for `text-decoration-line`](https://developer.mozilla.org/fr/docs/Web/CSS/text-decoration-line#compatibilit%C3%A9_des_navigateurs), there is no support for the property on IE at all, so `-ms-text-decoration-line` doesn't do anything.

Now they're gone entirely, we don't have to worry about it :)

Reply:
> But looking at the [support table for `text-decoration-line`](https://developer.mozilla.org/fr/docs/Web/CSS/text-decoration-line#compatibilit%C3%A9_des_navigateurs), there is no support for the property on IE at all, so `-ms-text-decoration-line` doesn't do anything.

my bad  ( * _ * )
> Now they're gone entirely, we don't have to worry about it :)

Yes indeed : )